allure-framework-modules/allure-selenium-module/src/main/java/com/capgemini/ntc/selenium/core/newDrivers/DriverExtention.java

```diff
@@ -337,6 +339,29 @@ public MenuElement elementMenu(By selector, By childsSelector, By subMenuSelecto
 		return new MenuElement(selector, childsSelector, subMenuSelector, childsSubMenuSelector);
 	}
 	
+	public HorizontalSliderElement elementHorizontalSlider(final By sliderContainerSelector) {
+		BasePage.getAnalytics()
+						.sendMethodEvent(BasePage.analitycsCategoryName);
+		return new HorizontalSliderElement(sliderContainerSelector);
+	}
+	
+	public HorizontalSliderElement elementHorizontalSlider(final By sliderContainerSelector, final By sliderSelector, final By valueSelector) {
```

Comment:
Just last thing :)   Please add java docs for "public" methods.

Reply:
All javadoc for DriverExtension methods is definded in INewWebDriver class. Other javadoc added to public methods as requested.